### DISCUSSION TITLE
after migration. not showing self-pooling plotnft

### DISCUSSION THREAD

**Discussion by nicezic** [Chia-Network/chia-blockchain#8347] [Support]
migration not showing self-pooling plotnft

I migrated to a new machine.

After node and wallet synced, 

'chia plotnft show'  says nothing neither in GUI.

plotnft info
===========
Launcher ID: eb689d7865885049a860dadf78de2a0bd578411c5be30c919b104abd3a871f5a
Target address (not for plotting): xch1wau964ulx59s6qs40pp090qquzrtz52mjpvple04kkuj90qa645qtmyywj
Owner public key: 8a09642a30b8c77ef40499a1b13e397509c33cd1176597b265ffc739a48fe5892abad7730ab1d8b09808757a7d8e7177
Pool contract address (use ONLY for plotting - do not send money to this address): xch18aklz2lxd628lnj2kpzh7ha49e7euuzwyfh5vy5th7fm596ea6es4ee0pv

**Comment by jack60612** [comment]
Looking up your launcher id

**Comment by BrandtH22** [comment]
@nicezic , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the newer version of the chia client will automatically populate pooling information (including self-pooling) when the wallet resyncs: https://www.chia.net/downloads/

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
